### PR TITLE
Form: make objects collapsible by default

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Change-type: minor
Signed-off-by: Lucian <lucian.buzzo@gmail.com>

### CODE REVIEW

**Comment by cyplo** [comment]
Interesting ! From the client point of view - we'd like the ability to define the `collapsed/-ible` component in the `jsonschema` definition. The definition is for `a thing that gets us data` - would that map directly to the `Form` component in rendition ? Would that be `accordion` + `something that wraps Form and accordion` ? Good question :)

**Comment by cyplo** [comment]
@sradevski @LucianBuzzo - any news on this one ?

**Comment by LucianBuzzo** [comment]
I need to update this to use the `Accordian` from Grommet https://v2.grommet.io/accordion

**Comment by LucianBuzzo** [comment]
collapsibleform

File: src/unstable/components/Form/templates/ObjectFieldTemplate.tsx [hunk 1]
```diff
@@ -1,26 +1,65 @@
...
+	const { DescriptionField, idSchema } = props;
+
+	// The `activeIndex` and `onActive` props are used to make the accordion
+	// a controlled input, allowing the accordion to default to being open
+	const [active, setActive] = React.useState(true);
+
+	const toggleActive = () => setActive(!active);
+
+	const title =
+		props.schema.title && (props.uiSchema['ui:title'] || props.title)
```
Comment (sradevski) [inline src/unstable/components/Form/templates/ObjectFieldTemplate.tsx]: I would rather have a "ui:collapsible" or a similar option, as we might not always want to have a collapsible form whenever there is a title.

File: src/theme.ts [hunk 2]
```diff
@@ -246,6 +248,18 @@ export default {
 		extend: 'border: none;',
 		color: colors.primary.main,
 	},
+	accordion: {
+		border: {
+			color: colors.quartenary.main,
+		},
+		icons: {
+			// collapse: <ChevronUp />,
```
Comment (sradevski) [inline src/theme.ts]: remove the comments.

**Pull request closed without merging**